Tests/APNSTests/APNSTests.swift

```diff
@@ -7,9 +7,11 @@ class APNSTests: XCTestCase {
         defer { app.shutdown() }
 
         app.apns.configuration = try .init(
-            keyIdentifier: "9UC9ZLQ8YW",
-            teamIdentifier: "ABBM6U9RM5",
-            signer: .init(buffer: ByteBufferAllocator().buffer(capacity: 1024)),
+            authenticationMethod: .jwt(
+                key: .private(filePath: "/Users/kylebrowning/Desktop/AuthKey_9UC9ZLQ8YW.p8"),
```

Comment:
We'll need a better way of loading this in

Reply:
Yeah we discussed dumping in a bunk cert that will validate locally, but fail to work against Apple.